--- a/src/Application.vala	2019-05-06 06:03:55.000000000 +0200
+++ b/src/Application.vala	2019-06-02 18:32:20.105244744 +0200
@@ -27,7 +27,7 @@
 public class Spice.Application : Granite.Application {
     public const string PROGRAM_NAME = N_("Spice-Up");
     public const string ABOUT_STOCK = N_("About Spice-Up");
-    public const string APP_ID = "com.github.philip-scott.spice-up";
+    public const string APP_ID = "com.github.philip_scott.spice-up";
     public const string RESOURCE_PATH = "/com/github/philip-scott/spice-up/";
 
     public bool running = false;
@@ -79,6 +79,8 @@
 
     private void init () {
         if (!running) {
+            Gtk.Settings.get_default().gtk_theme_name = "elementary";
+            Gtk.Settings.get_default().gtk_icon_theme_name = "elementary";
             Gtk.Settings.get_default ().gtk_application_prefer_dark_theme = true;
             weak Gtk.IconTheme default_theme = Gtk.IconTheme.get_default ();
             default_theme.add_resource_path (RESOURCE_PATH);
