--- a/src/Application.vala	2019-01-20 01:13:29.000000000 +0100
+++ b/src/Application.vala	2019-02-25 20:43:48.047551659 +0100
@@ -27,7 +27,7 @@
 public class Spice.Application : Granite.Application {
     public const string PROGRAM_NAME = N_("Spice-Up");
     public const string ABOUT_STOCK = N_("About Spice-Up");
-    public const string APP_ID = "com.github.philip-scott.spice-up";
+    public const string APP_ID = "com.github.philip_scott.spice-up";
     public const string RESOURCE_PATH = "/com/github/philip-scott/spice-up/";
 
     public bool running = false;
@@ -61,6 +61,10 @@
 
     public override void open (File[] files, string hint) {
         init ();
+      
+        Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
+        Gtk.Settings.get_default().set_property("gtk-icon-theme-name", "elementary");
+
         foreach (var file in files) {
             if (is_file_opened (file)) {
                 // Preset active window with file
