diff --git a/src/Application.vala b/src/Application.vala
index b23da71..c5c4787 100644
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -28,7 +28,7 @@ namespace Spice {
 public class Spice.Application : Granite.Application {
     public const string PROGRAM_NAME = N_("Spice-Up");
     public const string ABOUT_STOCK = N_("About Spice-Up");
-    public const string APP_ID = "com.github.philip-scott.spice-up";
+    public const string APP_ID = "com.github.philip_scott.spice-up";
     public const string RESOURCE_PATH = "/com/github/philip-scott/spice-up/";
 
     public bool running = false;
@@ -65,6 +65,7 @@ public class Spice.Application : Granite.Application {
 
             settings = Spice.Services.Settings.get_instance ();
             window = new Spice.Window (this);
+            Gtk.Settings.get_default().set_property("gtk-theme-name", "elementary");
             this.add_window (window);
 
             running = true;
