From 15899a28980a0a00aa03b4fc711db4fd21154bba Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Thu, 7 Sep 2023 19:22:31 +0530
Subject: [PATCH] Elementary theme patch

---
 src/Application.vala | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/Application.vala b/src/Application.vala
index e304d07..2fa907f 100644
--- a/src/Application.vala
+++ b/src/Application.vala
@@ -77,6 +77,8 @@ public class Spice.Application : Gtk.Application {

     private void init () {
         if (!running) {
+            Gtk.Settings.get_default().gtk_theme_name = "elementary";
+            Gtk.Settings.get_default().gtk_icon_theme_name = "elementary";
             Gtk.Settings.get_default ().gtk_application_prefer_dark_theme = true;
             weak Gtk.IconTheme default_theme = Gtk.IconTheme.get_default ();
             default_theme.add_resource_path (RESOURCE_PATH);
--
2.41.0
